crucible-mir/src/Mir/Trans.hs

```diff
@@ -2621,6 +2647,13 @@ transCollection col halloc = do
     hmap2 <- mkShimHandleMap col halloc
     let hmap = hmap1 <> hmap2
 
+    --forM_ (Map.toList hmap) $ \(name, mh) -> do
+    --  MirHandle _ sig handle <- return mh
+    --  traceM $ "function " ++ show name ++ ":"
+    --  traceM $ "  sig = " ++ show sig
+    --  traceM $ "  handle = " ++ show (FH.handleType handle)
```

Comment:
Perhaps we should always print this if the `debugLevel` (from `MirGenerator`) is high enough?

Reply:
Enabled for `-d4` and higher in a16f6f8a9d6cf2155fde79e6433f0f3408b44ba9